doc/source/io.rst

```diff
@@ -1175,6 +1175,80 @@ data that appear in some lines but not others:
     0  1  2   3
     1  4  5   6
     2  8  9  10
+    
+Handling "bad" lines - preserving the data
```

Comment:
Not sure if you need this.  I think one giant section will work.

Reply:
If we eliminate the ``open`` workaround, I agree. Otherwise it felt like the ``open`` workaround might be difficult to understand in context.

Reply:
Even still, you can add a transition sentence instead between your code-blocks, adding this `open` workaround is *also* for preserving data.